### PR TITLE
Noah stoeckel

### PR DESCRIPTION
**Pull request by BoxOfCereal**
PULL!

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1 @@
+JWT_SECRET=test
```
Comment (johnoro) [inline .env:1]: I know your secret(s); with real apps, you'd definitely want your .env in your .gitignore

File: users-folder/usersModel.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+module.exports = {
+  add,
+  find,
+  findByUsername,
+  findById,
+  findByDepartment
+};
+
+function find() {
+  return db("users").select("id", "username", "password");
```
Comment (johnoro) [inline users-folder/usersModel.js:12]: Why use `select` here? When you call `db('users')`, it already selects all of the fields by default:
```javascript
  return db('users');
```
except this is excluding the department now; is that on purpose?

**Review by johnoro — APPROVED** [review]
Great work!

File: .env (hunk 1 above)
Comment (BoxOfCereal) [reply .env:1]: Oh man! Thank you for reminding me!

File: users-folder/usersModel.js (hunk 2 above)
Comment (BoxOfCereal) [reply users-folder/usersModel.js:12]: No this was just a slip, thank you for the suggestion!

**Pull request merged**